bazel/repository_locations.bzl

```diff
@@ -936,17 +885,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         strip_prefix = "rules_rust-{version}",
         urls = ["https://github.com/bazelbuild/rules_rust/archive/{version}.tar.gz"],
         use_category = ["dataplane_ext"],
-        extensions = [
-            "envoy.access_loggers.wasm",
-            "envoy.bootstrap.wasm",
-            "envoy.filters.http.wasm",
-            "envoy.filters.network.wasm",
-            "envoy.stat_sinks.wasm",
```

Comment:
`rules_rust` is also used for building tests.

Reply:
test dependencies are not counted, this used to be `test_only` before we added wasmtime.

Reply:
FYI, https://github.com/envoyproxy/envoy/pull/13932/commits/3f62cc67141c6f3d229bc70ed737073087ca10ac#diff-a2ca502474bb8ec5f092176f7ee6d2f11fc395ebb00e1717d6fbad9b16ee8fd8L905 this commit changes `test_only` to `dataplane_ext`

Reply:
in wasmtime PR